ontology-assets/ontology_info.json

```diff
@@ -45,6 +45,16 @@
         "version": "v2023-05-18",
         "source": "https://github.com/pato-ontology/pato/raw",
         "filename": "pato.owl"
+      },
+      "FBbt": {
```

Comment:
instead of adding it to schema 5.2, should we instead make a new, experimental only "5.2.0+experimental" version (or similar if that doesn't follow semver properly) and include it there? its for local use for now so it wouldn't be a huge deal either way but it'll be less confusing for version management that way